src/core/caniuse.js

```diff
@@ -68,20 +70,25 @@ export async function run(conf) {
         `Couldn't find feature "${feature}" on caniuse.com? ` +
         "Please check the feature key on [caniuse.com](https://caniuse.com)";
       pub("error", msg);
-      const featureURL = "https://caniuse.com/#feat=" + feature;
       content = hyperHTML`<a href="${featureURL}">caniuse.com</a>`;
     }
     resolve(content);
   });
   const definitionPair = hyperHTML.bind(document.createDocumentFragment())`
-    <dt class="caniuse-title"
-      id="${`caniuse-${feature}`}">Can I Use this API?</dt>
+    <dt class="caniuse-title">Can I Use this API?</dt>
     <dd class="caniuse-stats">${{
       any: contentPromise,
       placeholder: "Fetching data from caniuse.com...",
     }}</dd>`;
   headDlElem.appendChild(definitionPair);
   await contentPromise;
+
+  // remove from export
+  pub("amend-user-config", { caniuse: undefined });
```

Comment:
Maybe set it to { caniuse: feature } ? - no need to totally trash it. Or is causing problems?

Reply:
not really causing problems but I didn't find it useful there. We need it?

{caniuse: feature} would be good. Keeping whole configuration there seems, so, unnecessary.

Reply:
Let's go with `{caniuse: feature}`